ui/src/components/IndividualCard.vue

```diff
@@ -69,6 +86,37 @@ export default {
       } else {
         return `mdi-${datasource}`;
       }
+    },
+    async toggleLockedStatus() {
+      if (this.$apollo) {
+        try {
+          const mutation = this.locked
+            ? unlockIndividual(this.$apollo, this.uuid)
+            : lockIndividual(this.$apollo, "def");
```

Comment:
There's an error in here. When clicking to `lock` an `Individual`, the function passes `def` instead of the `uuid`.

Reply:
Thanks for noticing, I fixed it.

Reply:
I see you have changed it. The reason why we didn't catch this before is because we're not validating the parameters we're passing to the mock functions. This validation should be done on the unit tests. Can you add some assertions? In Python there are several methods that do that, for example : https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_with . I guess there's something similar on `jest`.

Reply:
I added the assertions to the "Lock individual on click" and "Unlock individual on click" tests.